src/test/cypress/fixtures/programming_exercise_submissions/build_error/BubbleSort.txt

```diff
@@ -0,0 +1 @@
+this should fail
```

Comment:
Why even have this as a fixture?

Reply:
Because the `OnlineEditorPage` requires an `ProgrammingExerciseSubmission` in the `typeSubmission` method. A `ProgrammingExerciseSubmission` contains a list of pairs of file name and path. The file path points to a file, whichs content will be typed into the online editor. I did not want to create a new method for this one usecase.